docs/example/public-hub-content.mdx

```diff
@@ -168,11 +168,32 @@ helm upgrade capact --version {VERSION} capactio/capact -n capact-system --reuse
 3. Execute `populator`:
 
    ```bash
-    APP_JSONPUBLISHADDR=http://{your_ip_address} populator register ocf-manifests {PATH_TO_DIR_WITH_MANIFESTS}
+    APP_JSON_PUBLISH_ADDR=http://{your_ip_address} populator register ocf-manifests --source {PATH_TO_DIR_WITH_MANIFESTS}
    ```
 
    >**NOTE**: More information about using `populator`, can be found in [`populator_register-ocf-manifests.md`](https://github.com/capactio/capact/blob/main/cmd/populator/docs/populator_register-ocf-manifests.md).
 
+## Using many manifests sources
+
+You can provide multiple manifest sources by setting overrides using different indexes of the `hub-public.populator.manifestsLocations`.
```

Comment:
```suggestion
You can provide multiple manifest sources by setting overrides using different indexes of the `hub-public.populator.manifestsLocations`.

>**NOTE**: All manifest source locations must contain manifests with unique paths. In case of a path conflict, the Populator will fail before populating the Public Hub database.
```

^ Please make sure the NOTE specifies the actual behavior of Populator 🙂